app/Resources/views/admin/blog/_form.html.twig

```diff
@@ -6,7 +6,11 @@
     {{ form_start(form, { attr: { novalidate: 'novalidate' } }) }}
 #}
 
-{{ form_start(form) }}
+{% if show_confirmation_message is defined and show_confirmation_message == true %}
```

Comment:
javier do not worry, in this PR I learned a lot of comments. I am in favor of less code, more simple. fixed!